common/components/Footer/PreFooter.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './PreFooter.scss';
+
+const PreFooter = () => {
+  return (
+    <section className="pre-footer">
+      <div className="container">
+        <p>
+          MyEtherWallet.com does not hold your keys for you. We cannot access
+          accounts, recover keys, reset passwords, nor reverse transactions.
+          Protect your keys &amp; always check that you are on correct URL.
+          <a href="#"> You are responsible for your security.</a>
```

Comment:
We don't yet have the disclaimer modal, but when we do, it will pop up when this is clicked.

On a separate note, I'm working on a action/side-effect based modal instead of component state based modal popups today.

Reply:
So it would be like notifications? A saga would fire some kind of modal action? Wouldn't the modal need sufficiently complicated args for content / buttons?

Reply:
Yes, very similar to notifications.

Initially this would be a similar confirmation dialog with only a title, body, cancel and accept buttons to minimize complexity.